MetaMask/metamask-extension
# Update the changelog when creating an RC

The changelog will now be automatically updated when a release branch is created. A new release header along with changelog entries for any new commits will be added.

Note that this changelog will still need to be manually cleaned up, but it's one less manual step at least.

The old Bash script for adding a new release header to the changelog has been removed, as that functionality is now built into the changelog update script.

A new script has been added to commit any changes made to the manifest and changelog. This step used to happen at the end of the bump manifest version script, but now the changelog update relies upon the manifest version bump happening first, so it needed to be re-ordered. The changes should only be committed on the first run of the branch, as it's contingent upon the manifest changing (due to the version bump). Further changelog updates won't trigger new automatic commits.

Relates to #10752

Manual testing steps:  
  - Create a new release branch
  - See that it updates the changelog automatically
